codecov.yml

```diff
@@ -33,7 +33,7 @@ component_management:
     - component_id: query-codemods
       name: '@tanstack/query-codemods'
       paths:
-        - packages/codemods/**
+        - packages/query-codemods/**
```

Comment:
Only this package have different directory name with name field of package.json. so I matched it